### PR TITLE
testing: remove duplicate dependencies and superfluous builddir statements

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by maxice8** [comment]
CI can be ignored as no packages will be rebuilt, the changes do not affect the artifacts (code compiled) or the metadata (information like homepge) given to the user.

**Comment by ad-m** [comment]
Can we introduce tests so that such repetition of dependencies can be detected automatically on build?

**Comment by maxice8** [comment]
I have apkbuild-lint in maxice8/atools

I plan on having it be official one day

**Comment by maxice8** [comment]
correction, aports-lint is the binary that perform the duplicate dependencies check.

Anyways here is my git hook for running it on every template i am working on before comitting it: http://ix.io/1JtY

**Comment by maxice8** [comment]
You can also run it freely

`fd APKBUILD community/ -exec aports-lint '{}' \;`

File: testing/hitch/APKBUILD [hunk 1]
```diff
@@ -8,19 +8,17 @@ url="https://hitch-tls.org/"
 arch="all"
 license="BSD"
 makedepends="libev-dev openssl-dev automake autoconf py-docutils bison flex
-	autoconf automake libtool"
+	autoconf libtool"
```
Comment (TBK) [inline testing/hitch/APKBUILD]:
```suggestion
	libtool"
```

Comment (maxice8) [reply testing/hitch/APKBUILD]: fixed

**Pull request closed without merging**